tools/spectral/.spectral.yaml

```diff
@@ -172,9 +172,8 @@ rules:
       - field: additionalServices
         function: pattern
         functionOptions:
-          match: "^(mms)$"
-        message: "'additionalServices' must be 'mms' as no other services are supported."
-
+          match: "^mms(,[a-zA-Z0-9_-]+)*$"
+        message: "'additionalServices' must start with 'mms' and can include additional services in a comma-separated format."
```

Comment:
could you clarify this change? In which scenario we will have `mms,service_name,ABC`?

Reply:
as of now, we don't have other services, but SRE asked us to have this as a list and available for others. since I'm doing the PR for api-registry to return this, I'm adding some linting considering the comma separated names